content/client-server-api/modules/end_to_end_encryption.md

```diff
@@ -597,6 +597,30 @@ package naming convention.
 
 {{% event event="m.key.verification.cancel" %}}
 
+###### General key verification error handling
+
+At any point the interactive verification can go wrong. The following describes
+what to do when an error happens:
+
+-   Alice or Bob can cancel the verification at any time. An
+    `m.key.verification.cancel` message must be sent to signify the
```

Comment:
It would be great to turn these event types into links, while you're here:

```suggestion
    [`m.key.verification.cancel`](#mkeyverificationcancel) message must be sent to signify the
```